package.sh

```diff
@@ -213,9 +214,11 @@ check_tag_exists () {
 make_dir_tree() {
     work_dir=$1
     version=$2
-    mkdir -p $work_dir/$INSTALL_ROOT_DIR/versions/$version/scripts
+
+    mkdir -p $work_dir/$INSTALL_ROOT_DIR
```

Comment:
I agree we're not consistent in our checks. I'm going to be doing a larger revision to the package script to allow for multi-arch builds, so I'll work on improving that with the next batch of changes.